lib/tasks/data.rake

```diff
@@ -75,4 +76,137 @@ namespace :aleftng do
       end
     end
   end
+
+  def find_and_replace_in_text(find, replace, text)
+    #puts "Search and replace #{find.inspect} => #{replace.inspect}"
+    text.gsub!(find, replace)
+    return text
+  end
+
+  # TODO - Výnimka - iný typ otázky ako tieto základné
+  def find_question_type(text)
+    if text == "single-choice"
+      return "SingleChoiceQuestion"
+    elsif text == "multi-choice"
+      return "MultiChoiceQuestion"
+    elsif text == "answer-validator"
+      return "EvaluatorQuestion"
+    elsif text == "complement"
+      return "Complement"
+    end
+  end
+
+  def find_substring(text)
+    if text.include? "<correct>"
```

Comment:
- je nezmysel robit ```ak test == true, vrat true, inak (test == false), vrat false``` to je predsa to iste ako ```vrat test```
- metoda ma nezmyselny nazov, hladaj podretazec, ale aky? parameter to nie je a hlada sa konstanta ukryta vo vnutri metody

Nahradit priamo volanim toho include? v mieste pouzitia metody.